tests/test_script_queue.py

```diff
@@ -74,17 +74,20 @@ async def setup_types(self) -> None:
             .field_info
         )
         self.available_scripts.standard = (
-            "std_script1,std_script2,auxtel/std_script1,maintel/std_script1,"
+            "std_script1,std_script2:auxtel/std_script1,maintel/std_script1"
```

Comment:
Please use the constant from `script_queue.py` if you chose to adopt that recommendation. Also in lines 80, 166 and 181.